src/test/java/Task3package/Task3.java

```diff
@@ -0,0 +1,20 @@
+package Task3package;
+
+
+public class Task3 {
+    public static void main(String[] args) {
+
+        Сhef c = new Сhef();
+
+        c.add("Man", "pink", 1, 5);
+        c.add("Tam", "yellow", 2, 6);
+        c.add("Son", "brown", 3, 7);
+        c.add("Ban", "black", 4, 8);
+        c.add("Ban", "black", 4, 8);
```

Comment:
I think you would need to name them more comprehensible or even create a concrete objects for each vegetable type, because currently you're trying to pack different objects to one generalised object